tests/test_hausdorff_loss.py

```diff
@@ -0,0 +1,168 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+import unittest
+
+import numpy as np
+import torch
+from parameterized import parameterized
+
+from monai.losses import HausdorffDTLoss
+
+TEST_CASES = [
+    [  # shape: (1, 1, 2, 2), (1, 1, 2, 2)
```

Comment:
thanks for the PR, please help add some 3D and GPU test here, for example https://github.com/Project-MONAI/MONAI/blob/281cb0119c01eaa8e6c841880b91f92f45e8d7f7/tests/test_focal_loss.py#L27

Reply:
Thanks for the feedback, this has been addressed in commits [3163797](https://github.com/Project-MONAI/MONAI/pull/6994/commits/31637978cc817b8c085c7754fcf28f91ae3a515c) and [8c866de](https://github.com/Project-MONAI/MONAI/pull/6994/commits/8c866de1a15b625c8df444df50a71d9c5f08c3f4)